baseband/tests/test_core.py

```diff
@@ -63,6 +65,41 @@ def test_open_wrong_args():
     assert "inconsistent" in str(exc.value)
 
 
+def test_open_sequence(tmpdir):
+    """Test opening file sequences with `baseband.open`."""
+    # Open DADA file sequence by passing a list.
+    with baseband_open(SAMPLE_DADA) as fh:
+        data1 = fh.read()
+        header1 = fh.header0.copy()
+    header1.payload_nbytes = header1.payload_nbytes // 2
+
+    files = [str(tmpdir.join('f.{0:d}.dada'.format(x))) for x in range(2)]
+    with baseband_open(files, 'ws', format='dada', header0=header1) as fw:
+        fw.write(data1)
+
+    with baseband_open(files) as fn:
+        assert fn.info.format == 'dada'
+        assert len(fn.fh_raw.files) == 2
+        assert fn.header0 == header1
+        assert np.all(fn.read() == data1)
+
+    # Open VDIF file sequence by passing a FileNameSequencer.
+    files = sf.FileNameSequencer('f{file_nr:03d}.vdif', {})
+    with baseband_open(SAMPLE_VDIF) as fh:
+        data2 = fh.read()
+        header2 = fh.header0.copy()
+
+    with baseband_open(files, 'ws', format='vdif', nthread=8,
+                       file_size=(8 * header2.frame_nbytes), **header2) as fw:
```

Comment:
hey, total nitpick, but the parentheses are not helpful to me here... More relevant: maybe add a comment with how many frames the sample file contains.

Reply:
I think this is a more general difference in coding styles where I find something like `sample_rate=32*u.MHz` or `sample_rate=32 * u.MHz` more distracting than `sample_rate=(32 * u.MHz)`.  I'll switch to `file_size=8*header2.frame_nbytes` here and in other places.